.coveragerc

```diff
@@ -1,5 +1,6 @@
 [run]
 branch = True
+parallel = True
```

Comment:
Could you add a comment that this has been added as workaround and link to the original issue?